tools/travis_mvn_watchdog.sh

```diff
@@ -164,7 +164,7 @@ watchdog () {
 
 # Check the final fat jar for illegal artifacts
 check_shaded_artifacts() {
-	jar tf build-target/lib/flink-dist-*.jar > allClasses
```

Comment:
The problem is that our dist jar is called `flink-dist_2.10-1.3-SNAPSHOT.jar`. so the old variant didn't match the jar.
This leads to the following error in all travis build:
```
java.io.FileNotFoundException: build-target/lib/flink-dist-*.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:220)
	at java.util.zip.ZipFile.<init>(ZipFile.java:150)
	at java.util.zip.ZipFile.<init>(ZipFile.java:121)
	at sun.tools.jar.Main.list(Main.java:1060)
	at sun.tools.jar.Main.run(Main.java:291)
	at sun.tools.jar.Main.main(Main.java:1233)
``` 
I'm doing `jar tf` here to check if guava and other libraries are not part of the fat jar.